docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -55,6 +55,14 @@ We have added 4 new features to the flamegraph visualization:
 
 {{< video-embed src="/media/docs/grafana/panels-visualizations/screen-recording-grafana-10.1-flamegraph-whatsnew.mp4" >}}
 
+### Visualizations and Widgets split
+
+<!-- Alexa Vargas -->
+
+_Experimental in some of our free tiers of Grafana._
+
+This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

Comment:
@imatwawana I am not sure about this part, can you help me? 🙈 

>This experimental feature brings a clear distinction between two panel plugin types: visualizations and widgets. Now, easily add widgets like Text, News, and Annotation list, without selecting a data source.

In our current system, we don't have the concept of "widgets". We have different plugin types (Timeseries, Text, News, Stat panel), and some of them use data sources, while others don't need one. 
So, we're introducing a new concept called "widgets." Widgets are panels that don't require a data source. For example, we have widgets like the Text panel, News panel, and others.

I am wondering if this will bring more clarity:

```suggestion
This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualizations panels that consume a data source and a new type called "widgets" that don't require a data source. Now, users can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on widget or visualization type, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

Also, I want to add this screenshot, do you how do I do that? 

![WidgetVizSplit](https://github.com/grafana/grafana/assets/239999/0c5cb139-04f1-4ffd-a584-787b73f3567c)

Reply:
Yes that's much clearer. A couple tweaks:
```suggestion
This experimental feature introduces a clear distinction between two different categories of panel plugin types: visualization panels that consume a data source and a new type called _widgets_ that don't require a data source. 

Now, you can easily add widgets like Text, News, and Annotation list without the need to select a data source. The plugins list and library panels are filtered based on whether you've selected a widget or visualization, providing a streamlined editing experience. Early access to this exciting update will be granted to some of our Cloud free tier users.
```

Reply:
Upload your image [here](https://admin.grafana.com/upload/media/docs/grafana/dashboards/) and then add in the reference to the image location starting the file path at "media". If you want to control the width of the image, then use the [`figure` shortcode](https://grafana.com/docs/writers-toolkit/write/shortcodes/#figure-shortcode) instead of regular markdown to add the image. The `figure` shortcode has a max-width property that you can set.